.gitignore

```diff
@@ -1,9 +1,8 @@
 node_modules/
-.env
 .nyc_output
 .vscode/
 coverage
-
+.gitignore
```

Comment:
You should not `.gitignore`, `.gitignore`

Reply:
removed the. gitignore file